.env

```diff
@@ -0,0 +1,19 @@
+# This file defines all environment variables that the application needs.
+# DO NOT DEFINE PRODUCTION SECRETS IN THIS FILE.
+# Use ".env.local" for local overrides during development.
+# Use real environment variables when deploying to production.
+# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
+
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_SECRET=CHANGEME
```

Comment:
This isn't a .env.dist?

Reply:
.env handling best practice was changed by Symfony just last week: https://symfony.com/blog/improvements-to-the-handling-of-env-files-for-all-symfony-versions